app/controllers/apis/ScratchController.scala

```diff
@@ -21,21 +21,24 @@ import javax.inject.{Inject, Singleton}
 import models.errors.InvalidProcessError
 import play.api.libs.json.{JsValue, Json}
 import play.api.mvc._
-import services.ScratchService
+import services.{ScratchService, AuditService}
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
-
+import models.audit.DummyScratchSubmissionEvent
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 @Singleton
-class ScratchController @Inject() (appConfig: AppConfig, scratchService: ScratchService, mcc: MessagesControllerComponents) extends FrontendController(mcc) {
+class ScratchController @Inject() (appConfig: AppConfig, scratchService: ScratchService, auditService: AuditService, mcc: MessagesControllerComponents)
+    extends FrontendController(mcc) {
 
   implicit val config: AppConfig = appConfig
 
   def submitScratchProcess(): Action[JsValue] = Action.async(parse.json) { implicit request: Request[JsValue] =>
     scratchService.submitScratchProcess(request.body).map {
       case Right(submissionResponse) =>
         val location: String = s"/guidance/scratch/${submissionResponse.id}"
+        // TODO Dummy event log, as currently auditable code incomplete
```

Comment:
Delete the `TODO` and use one of the *real* audit models. This means the `DummyScratchSubmissionEvent` is no longer needed.